ts/torch_handler/image_classifier.py

```diff
@@ -18,13 +32,28 @@ class ImageClassifier(VisionHandler):
     topk = 5
     # These are the standard Imagenet dimensions
     # and statistics
-    image_processing = transforms.Compose([
-        transforms.Resize(256),
-        transforms.CenterCrop(224),
-        transforms.ToTensor(),
-        transforms.Normalize(mean=[0.485, 0.456, 0.406],
-                             std=[0.229, 0.224, 0.225])
-    ])
+    
+    if not ipex_enabled:
```

Comment:
why are there 2 code paths for image preprocessing?

Reply:
I've just made the code change so that the _optimized_preprocessing is only applied when ipex_enable=true. But this optimization is generalizable to without ipex as well. We do not need 2 code paths.